### DISCUSSION TITLE
[Question] How to use navmesh?

### DISCUSSION THREAD

**Discussion by yizhengzhang1** [isaac-sim/IsaacLab#1108] [Q&A]
### Question

Hi,

I want to use orbit framework to collect some data and need the robot in the isaacsim navigate with collision free path.

The navmesh seems a good way to do it,
I want to know how to add navmesh in the orbit framework.

Can you give an example using navmesh?

I also notice that the windows using standalone python scripts does not contain **navigation** choice in **Windows menu** while the origin isaacsim ui has.

The document of navmesh :https://docs.omniverse.nvidia.com/extensions/latest/ext_navigation-mesh/api.html
does not contain how to set agent config like agent height, agent radius and voxel ceiling.

**Comment by Dhoeller19** [comment]
Hi @Privilger, unfortunately, we are not planning to support this currently.
